src/components/link/Link.tsx

```diff
@@ -0,0 +1,41 @@
+import Image from 'next/image'
+
+type Icon = {
+  leftIcon?: string
+  rightIcon?: JSX.Element
+  width?: number
+  height?: number
+}
+
+type LinkProps = {
+  url?: string
+  name: string
+  icon?: Icon
+}
+
+const Link: React.FC<LinkProps> = ({ url, name, icon }): JSX.Element => (
+  <a
+    className="okp4-nemeton-web-page-link-main"
+    href={url}
+    key={name}
```

Comment:
That's weak, it is not at all forbidden to have a `link` with the same `name` but which refers to a different namespace, resource or url.